examples/Create.php

```diff
@@ -36,48 +36,53 @@
 // Here we use the WebService to get the schema of "customers" resource
 try
 {
-	$webService = new PrestaShopWebservice(PS_SHOP_PATH, PS_WS_AUTH_KEY, DEBUG);
-	$opt = array('resource' => 'customers');
-	if (isset($_GET['Create']))
-		$xml = $webService->get(array('url' => PS_SHOP_PATH.'/api/customers?schema=blank'));
-	else
-		$xml = $webService->get($opt);
-	$resources = $xml->children()->children();
-}
-catch (PrestaShopWebserviceException $e)
-{
-	// Here we are dealing with errors
-	$trace = $e->getTrace();
-	if ($trace[0]['args'][0] == 404) echo 'Bad ID';
-	else if ($trace[0]['args'][0] == 401) echo 'Bad auth key';
-	else echo 'Other error<br />'.$e->getMessage();
+    $webService = new PrestaShopWebservice(PS_SHOP_PATH, PS_WS_AUTH_KEY, DEBUG);
+    $opt = array('resource' => 'customers');
+    if (isset($_GET['Create']))
+        $xml = $webService->get(array('url' => PS_SHOP_PATH.'/api/customers?schema=blank'));
+    else
+        $xml = $webService->get($opt);
+    $resources = $xml->children()->children();
+
+} catch (PrestaShopWebserviceNotFoundException $exception) {
+    echo 'Bad ID';
```

Comment:
Just a suggestion but maybe this could be implemented by the `PrestaShopWebserviceException`, maybe not `getMessage` but an alternative one like `getApiErrorMessage` this way you could simplify this try/catch

```php
try {
   ...
} catch (PrestaShopWebserviceException $e) {
   echo $e->getApiErrorMessage();
   exit;
}
```

And leave all the logic about error messages in the exception instead of duplicating it here The base class can implement a generic message base on `$this->getMessage` while the specific children classes can override the method with a custom message

But it's just a suggestion, not mandatory if you don't think it's relevant